README.md

```diff
@@ -32,34 +33,51 @@ Explanations for commands.
 - `nvm use` : Install and use the correct version of Node.
 - `npm i` : As stated above; Install dependencies and link local packages.
 
+Related files.
+- `.nvmrc` : Defines the node version used to compile the theme.
+- `package.json and package-lock.json` : Defines the node modules for compiling the theme.
+- `postcss.config.js and postcss.plugins.js` : Configurations for the postcss-tool that is run when the theme is built.
+You can read more about the tool here: https://postcss.org/
+- `webpack.config.js` : Configuration file for the webpack-tool that is used to actually build the theme. Similar tool
+to Gulp or Grunt. Usually if there is something wrong with the compiled theme files the culprit can be found here.
+- `webpack.svgToSprite.js` : This file is used to create a sprite of all the icons used on the site. It gets all the
+icons from the src-folder, compiles them into a sprite under `dist/icons` and also creates a css-file where the icons are
+referenced called _hdbt-icons.css_.
+
 ## Structure for files and folders
 
-```
-hdbt
-│   README.md
-└───src
-│   └───scss
-│   │   │   styles.scss
-│   └───js
-│   │   │   common.js
-│   └───icons
-│       |   sprite.svg
-│       └───subdir
-│           |   some-icon.svg
-└───dist
-    └───css
-    └───js
-    └───icons
-```
+### The config-folder
```

Comment:
> The whole chapter `Structure for files and folders`

Suggestion: 
### The config folder
The config folder includes configurations that are used when installing a new project from scratch. These configuration files are copied under the `conf/cmi` folder and used there. Therefore, altering them under the theme doesn't change anything unless you are building a new instance.

### The dist and src folders
Under the `./src` folder, there are all the theme components that are being compiled to the `./dist` folder, such as CSS, JavaScript, icons, and fonts. The `./dist` folder includes the compiled version of the same information created using the commands listed under the commands title.

The theme styles under the `./scss` folder are structured by implementing principles from the ITCSS architecture but with small adjustments to make it work for the needs of the project.

### Templates folder
Under the `./templates` folder, the structure is similar to the base-theme stable9 with a few additions, such as the `./templates/module` folder that includes templates for the helfi-prefixed modules created for this project.

### Translations folder
The `./translations` folder includes translations for all the translatable strings provided by the hdbt-theme.

Reply:
These were some good changes! Thank you! Fixed! 🦖